src/api/parser/src/utils/indexer.js

```diff
@@ -0,0 +1,174 @@
+const { setIntervalAsync, clearIntervalAsync } = require('set-interval-async/dynamic');
+
+const { ELASTIC_MAX_RESULTS_PER_PAGE = 5, POSTS_URL = 'http://localhost/v1/posts' } = process.env;
+const { logger } = require('@senecacdot/satellite');
+const { Elastic } = require('@senecacdot/satellite');
```

Comment:
```suggestion
const { logger, Elastic } = require('@senecacdot/satellite');
```

Reply:
Thanks for this, I went back and look for all similar imports